### DISCUSSION TITLE
Creating multi-output activity

### DISCUSSION THREAD

**Discussion by LUCASmaxime** [LCA-ActivityBrowser/activity-browser#996] [Q&A] [answered]
Hi everyone,
I would like to create a multi-ouput activity under Activity-browser, and use impact allocation factors.
Could anyone advice how to do so?
Many thanks!
Best,
Maxime

**Comment by marc-vdm** [comment, marked as answer]
Hi Maxime, There is currently no 'official' way to create multifunctional processes in AB, but you can create them by creating multiple processes. One 'abstract process' with all the inputs/waste outputs and elementary flows that creates '1 unit' representing all products together, and then one process for each output product. Those output processes then only use some fraction of the input from the abstract process.

The database below has a few examples:

[MF_example_DB.xlsx](https://github.com/LCA-ActivityBrowser/activity-browser/files/12012727/MF_example_DB.xlsx)

**Reply by LUCASmaxime** [reply]
Great! Many thanks for the reactive support and the example.
Best
Maxime
